README.md

```diff
@@ -10,9 +10,9 @@ _Staging:_
 [![Trees-GitHub Issues](https://img.shields.io/github/issues/nciinc/fs-permit-platform.svg)](https://github.com/nciinc/fs-permit-platform/issues)
 [![Code Climate](https://codeclimate.com/github/nciinc/fs-permit-platform/badges/gpa.svg)](https://codeclimate.com/github/nciinc/fs-permit-platform)
 
-# U.S. Forest Service Intake Module
+# U.S. Forest Service Permit Platform
 
-** Module for intake of special use applications for Forest Service Application Permits. **
+** Platform for sale of Christmas tree permits and intake of special use applications for the Forest Service **
```

Comment:
US Forest Service seems to be what they use on their web sites

Reply:
I'll update